src/components/recipes/loop/recipe.vue

```diff
@@ -16,7 +19,8 @@ export default {
   name: 'recipe',
   props: {
     recipe: {},
-    selectedIngredients: {}
+    selectedIngredients: {},
+    image: Boolean
```

Comment:
ik zou het "hasImage" genomd hebben is wat duidelijk dat het een boolean is dan

Reply:
yep is wss beter, zal het even veranderen